frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -37,117 +34,90 @@ end
+                prop = util.splitToArray(prop, "\n")
+                for i = 1, #prop do
+                    prop[i] = BD.auto(prop[i])
+                end
+                prop = table.concat(prop, "\n")
+            else
+                prop = BD.auto(prop)
+            end
+        elseif prop_key == "series" then
+            -- If we were fed a BookInfo book_props (e.g., covermenu), series index is in a separate field
+            if book_props.series_index then
+                -- Here, we're assured that series_index is a Lua number, so round integers are automatically
+                -- displayed without decimals
+                prop = prop .. " #" .. book_props.series_index
+            else
+                -- But here, if we have a plain doc_props series with an index, drop empty decimals from round integers.
+                prop = prop:gsub("(#%d+)%.0+$", "%1")
+            end
+        elseif prop_key == "language" then
+            -- Get a chance to have title, authors... rendered with alternate
+            -- glyphs for the book language (e.g. japanese book in chinese UI)
+            values_lang = prop
+        elseif prop_key == "description" then
+            -- Description may (often in EPUB, but not always) or may not (rarely in PDF) be HTML
+            prop = util.htmlToPlainTextIfHtml(prop)
         end
-        keywords = table.concat(keywords, "\n")
-    else
-        keywords = BD.auto(keywords)
+        table.insert(kv_pairs, { prop_text, prop })
     end
-    table.insert(kv_pairs, { _("Keywords:"), keywords })
-
-    local description = book_props.description
-    if description == "" or description == nil then
-        description = _("N/A")
-    else
-        -- Description may (often in EPUB, but not always) or may not (rarely
-        -- in PDF) be HTML.
-        description = util.htmlToPlainTextIfHtml(book_props.description)
-    end
-    -- (We don't BD wrap description: it may be multi-lines, and the value we set
-    -- here may be viewed in a TextViewer that has auto_para_direction=true, which
-    -- will show the right thing, that'd we rather not mess with BD wrapping.)
-    table.insert(kv_pairs, { _("Description:"), description })
-
+    local is_doc = self.document and true or false
     local viewCoverImage = function()
         self:onShowBookCover(file)
     end
-    table.insert(kv_pairs, { _("Cover image:"), _("Tap to display"), callback=viewCoverImage })
+    table.insert(kv_pairs, { _("Cover image:"), _("Tap to display"), callback=viewCoverImage, separator=is_doc })
 
-    -- Get a chance to have title, authors... rendered with alternate
-    -- glyphs for the book language (e.g. japanese book in chinese UI)
-    local values_lang = nil
-    if book_props.language and book_props.language ~= "" then
-        values_lang = book_props.language
+    -- Page section
+    if is_doc then
+        local lines_nb, words_nb = self:getCurrentPageLinesWordsNumber()
```

Comment:
@Frenzie : what's the grammatically correct name for this function ? (again, singular/plural... :/)
```
LineWordNumbers
LinesWordsNumbers
LineWordNumber
LinesWordsNumber
```

Reply:
An option was `getCurrentPageNumberOfLinesWords`.

Reply:
Probably LineWordCount?

Reply:
getCurrentPageLineWordCount or ...LineWordCount**s** (as it returns 2 numbers) ?

Reply:
Yes, good point. Let's go with that. ^_^